.gitignore

```diff
@@ -49,3 +49,4 @@ assets
 release/*
 release/logs
 debug
+tests/
```

Comment:
Why? These should be in git.

Reply:
I have a local folder (for fast testing) with currently 1 ipynb that's not relevant to add to the github

Reply:
OK, the name's too generic though. Maybe `.local-tests/` or `.private-tests/`?

Reply:
Actually, could you add it to our tests? This is exactly what tests are for